README.md

```diff
@@ -18,6 +21,20 @@ Livros recomendados pelos participantes das comunidades DevAraraquara / OpenSanc
 * **Comentários:**
     * Um livro que deve estar presente com todos desenvolvedores que prezam por qualidade de software. Um livro com ótimos conselhos e exemplos de como e por que devemos escrever um código limpo e como evitar os "maus cheiros" de nosso código.
 
+## Programação Nível Básico
+
+### Ruby
```

Comment:
Yes, that's one way of solving it. It'll be repetitive though, since it's under `## Programação Nível Básico`. If you say that's a go anyways, I'll change the heading to that. By the way, is it possible to append a commit to this pull request or should I close this and make a new request?